server/src-lib/Hasura/Incremental/Internal/Dependency.hs

```diff
@@ -16,11 +16,13 @@ import           Data.Functor.Classes          (Eq1 (..), Eq2 (..))
 import           Data.GADT.Compare
 import           Data.Int
 import           Data.Scientific               (Scientific)
+import           Data.Time.Clock
 import           Data.Vector                   (Vector)
 import           GHC.Generics                  ((:*:) (..), (:+:) (..), Generic (..), K1 (..),
                                                 M1 (..), U1 (..), V1)
 
 import           Hasura.Incremental.Select
+import           System.Cron.Types
```

Comment:
Nitpick, but this import belongs in the above import block, since it’s from a dependency, not graphql-engine.